tests/api_connexion/endpoints/test_dag_run_endpoint.py

```diff
@@ -1162,16 +1162,19 @@ def test_should_raises_403_unauthorized(self):
 
 
 class TestPatchDagRunState(TestDagRunEndpoint):
-    @pytest.mark.need_serialized_dag(True)
     @pytest.mark.parametrize("state", ["failed", "success"])
-    @freeze_time(TestDagRunEndpoint.default_time)
-    def test_should_respond_200(self, state, dag_maker, session):
+    def test_should_respond_200(self, state, session):
         dag_id = "TEST_DAG_ID"
         dag_run_id = 'TEST_DAG_RUN_ID'
-        with dag_maker(dag_id):
-            DummyOperator(task_id='task_id')
-        dr = dag_maker.create_dagrun(run_id=dag_run_id)
+        date = datetime(2021, 1, 1)
+        dag = DAG(dag_id, schedule_interval=None, start_date=date)
+        task = DummyOperator(task_id='task_id', dag=dag)
+        dag.tasks.append(task)
```

Comment:
Why is this needed?

Reply:
Removed. Added when I couldn't find why dag was not having tasks.